### PR TITLE
Translate attribute "uploaded" in MS

### PR DESCRIPTION
**Pull request by kamaroolkarim**

### CODE REVIEW

File: src/ms/validation.php [hunk 1]
```diff
@@ -112,23 +112,5 @@
     |
      */
 
-    'attributes'           => [
```
Comment (caouecs) [inline src/ms/validation.php]: Why destroy it ?

Comment (kamaroolkarim) [inline src/ms/validation.php]: actually, it was from my previous pull request. it was from my previous project. i forgot to remove it. sorry for that.

Comment (kamaroolkarim) [inline src/ms/validation.php]: ok done translate attribute "file"

**Comment by caouecs** [comment]
Thanks

**Comment by kamaroolkarim** [comment]
you are welcome bro 👌🏻

**Pull request merged**